src/decanter/core/jobs/data_setup.py

```diff
@@ -42,18 +41,17 @@ class DataSetup(DataUpload, Job):
         completed_at (str): The time the data was completed at.
         name (str): Name to track Job progress, will give default name if None.
     """
-    def __init__(self, train_data=None, setup_params=None, name=None):
+    def __init__(self, setup_input=None, name=None):
         """DataSetup Init.
 
         Args:
             train_data (:obj:`~decanter.core.jobs.data_setup.DataSetup)
             setup_params (:obj:`dict`, optional): Settings for setup data.
             name (:obj:`str`, optional): Name to track Job progress
         """
```

Comment:
Need to update all document to fit setup data

Reply:
done